README.md

```diff
@@ -27,7 +27,7 @@ $ nodejs app.js [PORT]
 
 Meaning of positional parameters:
 
-1. `PORT`: where Specberus will be listening for HTTP connections.  
```

Comment:
These two extra spaces at the end of the line weren't a typo. They're [a line break](http://spec.commonmark.org/0.15/#hard-line-breaks), so that default values appear neatly indented below the description of their argument.

But never mind, it's a silly thing :) It doesn't block your PR.